sgt-back-end/index.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable indent */
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const express = require('express');
+const app = express();
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Listening on port 3000!');
+});
+
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select *
+      from "grades"
+    `;
+  db.query(sql)
+    .then(result => {
+      res.json(result.rows);
+      res.status(200);
```

Comment:
The default status code is 200 and by the time line 28 runs, it's already been sent, so you can remove line 28.